test/unit/app/controllers/transactions/pending-tx-tracker-test.js

```diff
@@ -19,7 +20,7 @@ describe('PendingTransactionTracker', function () {
         getCompletedTransactions: sinon.stub().returns([]),
         approveTransaction: sinon.spy(),
         publishTransaction: sinon.spy(),
-        confirmTransaction: sinon.spy(),
+      confirmTransaction: sinon.spy(),
```

Comment:
Nit: the indentation is off here
```suggestion
        confirmTransaction: sinon.spy(),
```